### PR TITLE
Updated Effective Area IDL Calculation Text File

### PR DESCRIPTION
**Pull request by joyvelasquez**
This PR updates the original IDL Effective Area text files used for testing XRTpy testing to align with the latest modified IDL linear interpolation method. These updates ensure the test files reflect the new Effective Area calculations, improving accuracy and consistency.

### CODE REVIEW

**Comment by nabobalis** [comment]
Should the tests pass for these files changes (assuming the failures are due to that)?

**Comment by joyvelasquez** [comment]
Hi @nabobalis 

Updating these tests and revising the code method for the interruption tolerance value should help us pass before moving forward with the merge

File: xrtpy/response/tests/test_effective_area.py [hunk 1]
```diff
@@ -99,7 +101,7 @@ def test_EffectiveArea_exception_is_raised(name, date):
 def get_IDL_data_files():
     filter_data_files = []
     for dir in get_pkg_data_filenames(
-        "data/effective_area_IDL_testing_files", package="xrtpy.response.tests"
```
Comment (nabobalis) [inline xrtpy/response/tests/test_effective_area.py:102]: Is the plan to keep the old testing files?

File: xrtpy/response/tests/test_effective_area.py [hunk 2]
```diff
@@ -128,5 +130,5 @@ def test_effective_area_compare_idl(filename):
     assert u.allclose(
         actual_effective_area,
         IDL_effective_area,
-        rtol=1e-6,
+        rtol=1e-4, #Moderate Precision
```
Comment (nabobalis) [inline xrtpy/response/tests/test_effective_area.py:133]: Does it fail with -5 or -6?

File: xrtpy/response/tests/test_effective_area.py (hunk 1 above)
Comment (joyvelasquez) [reply xrtpy/response/tests/test_effective_area.py:102]: No, I plan to remove them soon

File: xrtpy/response/tests/test_effective_area.py (hunk 2 above)
Comment (joyvelasquez) [reply xrtpy/response/tests/test_effective_area.py:133]: Currently testing different values

**Comment by nabobalis** [comment]
This fixes https://github.com/HinodeXRT/xrtpy/issues/292 right?

**Comment by joyvelasquez** [comment]
Yes, I'll make sure to reference the issue to the PR and close the issue once I merge this PR

**Comment by jslavin** [comment]
My pull request (#310) supercedes this one.

**Pull request closed without merging**